source/declarations.tex

```diff
@@ -5429,7 +5432,7 @@
 the initializer expression is implicitly converted to a prvalue
 of type ``\cvqual{cv1} \tcode{T1}''.
 The temporary materialization conversion is applied and the reference is
-bound to the result.
+bound to the object to which the resulting xvalue refers.
```

Comment:
I think this was already correct. The result of a conversion is the entity to which the evaluation of the conversion refers.

Reply:
Ok, I'll make the change

Reply:
@zygoloid The result of a conversion is defined in terms of value categories, meaning that it's an expression as stated in [conv] p6.

Reply:
When we talk about expressions, we often mean "expression or conversion". That is the case when we define "result" as well as when we defined value categories. I continue to think that this wording is correct as-is, though wording elsewhere may need updating.